### ISSUE TITLE
Styling?

### ISSUE THREAD

**Issue by rafaqz** [JuliaGizmos/Interact.jl#194]
Is there a simple way to add css class/id/style tags to a widget?

**Comment by JobJob** [comment]
No, would be nice if we could support the layout options here: http://ipywidgets.readthedocs.io/en/stable/examples/Widget%20Styling.html

Not sure if they offer a way to set class/id tags.

Adding a layout option might be doable by adding something like this to the `view_state` function https://github.com/JuliaGizmos/Interact.jl/blob/master/src/IJulia/setup.jl#L227

```
haskey(widget_layouts, w) && state[:layout] = widget_layouts[w]

# and somewhere above that in setup.jl
widget_layouts = Dict{Widget, Dict{Symbol, Any}}()
```

You'd have to check the way ipywidgets expects those things - you can try them in python and check the websocket messages in the browser console. 

Disclaimer: It may not be as simple as that, but it's probably worth a shot

Also you might want to check/modify how hbox, vbox works, since it also uses layouts (in a slightly kludgy way) - see https://github.com/JuliaGizmos/Interact.jl/blob/master/src/IJulia/statedict.jl#L25

**Comment by rafaqz** [comment]
seems that ipywidgets has is a separate .style attribute for appearance, while .layout is for size related thing. But I it's the same idea

**Comment by cancandan** [comment]
How to do styling?
